MLStudio/ControlModel.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * @ Author: Gonzalez Dorian
+ * @ Description: Control Model class
+ */
+
+#include "ControlModel.hpp"
+
```

Comment:
Il faut ajouter le constructeur avec cette ligne :    
QQmlEngine::setObjectOwnership(this, QQmlEngine::ObjectOwnership::CppOwnership);